core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2702,6 +2705,32 @@ pub fn center_window(window: &Window, window_size: TaoPhysicalSize<u32>) -> Resu
   }
 }
 
+fn parse_proxy(url: &str) -> Result<ProxyConfig> {
+  let proxy = Url::parse(url).unwrap();
+
+  if proxy.scheme() == "http" && proxy.path() == "" {
```

Comment:
why are you checking for `proxy.path() == ""`?

Reply:
It depends on wry implementation. In wry [struct ProxyEndpoint](https://github.com/tauri-apps/wry/blob/dev/src/proxy.rs#L2), only host and port are supported, therefore I restrict here.

```rust
#[derive(Debug, Clone)]
pub struct ProxyEndpoint {
  /// Proxy server host (e.g. 192.168.0.100, localhost, example.com, etc.)
  pub host: String,
  /// Proxy server port (e.g. 1080, 3128, etc.)
  pub port: String,
}
```

Reply:
We can still remove this check and extract `host` and `port` from URL and pass it to wry